app/ide-desktop/lib/client/paths.ts

```diff
@@ -5,6 +5,10 @@ import * as utils from '../../utils.js'
 /** Path to the Project Manager bundle within the electron distribution (relative to the electron's resources directory). */
 export const PROJECT_MANAGER_BUNDLE = 'enso'
 
+export const SOURCE_FILE_EXTENSION = '.enso'
+
+export const BUNDLED_PROJECT_EXTENSION = '.enso-project'
```

Comment:
I don't think this is a good place to keep these constants. As described, "This module defines paths within the client distribution's resources.". I'd keep these in a separate module dedicated to opening URLs functionality.

Reply:
I will move this to the other project management stuff.